syntax = "proto3";

package web_transfer_api;

service WebTransferApi {
  rpc makeWebTransfer (IncomingWebTransfer) returns (OutgoingWebTransfer){}
}

message IncomingWebTransfer {
  string code = 1;
  string data = 2;
}

message OutgoingWebTransfer {
  bool success = 1;
  string data = 2;
  Error2 error = 3;
}

message Error2 {
  string code = 1;
  string data = 2;
}